gateway-sp-comms/src/single_sp.rs

```diff
@@ -1070,6 +1070,20 @@ impl AttachedSerialConsoleSend {
 
         rx.await.unwrap()
     }
+
+    pub async fn send_break(&self) -> Result<()> {
+        let (tx, rx) = oneshot::channel();
+        self.inner_tx
+            .send(InnerCommand::Rpc(RpcRequest {
+                kind: MgsRequest::SerialConsoleBreak,
+                our_trailing_data: None,
+                response_tx: tx,
+            }))
+            .await
+            .unwrap();
+        rx.await.unwrap();
+        Ok(())
+    }
 }
```

Comment:
We should also add a `SingleSp::send_host_nmi()` method and a corresponding `faux-mgs` subcommand to call it. It doesn't necessarily have to be a part of this PR but might make it more convenient to test.

Reply:
Yup, that was in my TODO list above; done in 89abdb393380383ef3c30e6a6e13757510eec73d

Reply:
Ah sorry, missed the TODO list. My bad.